aldryn_people/models.py

```diff
@@ -111,7 +117,11 @@ class Meta:
         verbose_name_plural = _('People')
 
     def __str__(self):
-        return self.name
+        pkstr = str(self.pk)
+
+        if six.PY2:
+            pkstr = six.u(pkstr)
+        return self.name.strip() or pkstr
```

Comment:
I really think you should explicitly check the return value of strip. Like this it's quite confusing to read.